packages/api/client.ts

```diff
@@ -372,6 +377,9 @@ export const getServiceStops = async (dbClient: Kysely<Database>, input: Service
         .where("fromStop.stopType", "not in", ignoredStopTypes)
         .where("toStop.stopType", "not in", ignoredStopTypes)
         .where((qb) => qb.where("fromStop.status", "=", "active").orWhere("toStop.status", "=", "active"))
+        .$if(!!input.modes?.[0], (qb) => qb.where("mode", "in", input.modes ?? ["---"]))
```

Comment:
should be `services.mode` instead of `mode` and use the fully qualifying name

Reply:
done